app/src/main/java/io/github/droidkaigi/confsched2018/presentation/sessions/SessionsViewModel.kt

```diff
@@ -34,6 +34,10 @@ class SessionsViewModel @Inject constructor(
 
     @OnLifecycleEvent(Lifecycle.Event.ON_CREATE)
     fun onCreate() {
+        refreshSessions()
+    }
+
+    fun refreshSessions() {
```

Comment:
Please create `onRetrySessions()` ↩️ 
I want to separate intention and action. 

```
...
fun onCreate() {
       refreshSessions()
}
fun onRetrySessions() {
       refreshSessions()
}
private fun refreshSessions() {

```

Reply:
No, Problem